pages/application.js

```diff
@@ -0,0 +1,113 @@
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+    faFacebook,
+    faInstagram,
+    faLinkedinIn,
+    faMediumM,
+    faTwitter,
+    faYoutube,
+  } from '@fortawesome/free-brands-svg-icons';
+import Head from 'next/head';
+
+import JoinUs from '../components/JoinUs';
+import { BackgroundColor } from '../components/Background';
+import NavBar from '../components/NavBar';
+import fireDb from '../utilities/firebase';
+
+const TryingContainer = styled.div` 
+  width: 100%;
+  height: 0;
+  background: url(/assets/trying.svg); 
+  background-repeat: no-repeat; 
+  background-position: right; 
+  background-size: contain; 
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding-bottom: 53%; 
+
+  ${(p) => p.theme.mediaQueries.tablet} {
+    top: 10%;
+  }
+
+  ${(p) => p.theme.mediaQueries.mobile} {
+    top: 42%;
+    padding-bottom: 100%;
+  }
+`; 
+
+const FooterContainer = styled.div`
+  text-align: center;
+  padding: 64px 0;
+  width: 100%;
+  color: ${p => p.theme.colors.secondary};
+`;
+
+const SocialMediaIcons = styled.div`
+  font-size: 32px;
+  margin-bottom: 16px;
+
+  a {
+    margin: 0 32px;
+  }
+`;
+
+const FullScreenBackgroundColor = styled(BackgroundColor)`
+  min-height: 120vh;
```

Comment:
Was there a reason for this? It leaves some extra space on some viewports

Reply:
sometimes the background div for the graphics would extend beyond 100vh, so I have it as 120vh to cover the image in all circumstances. Let me know if you want me to change it/remove it